tests/trufflehog_test.py

```diff
@@ -56,11 +56,10 @@ def testSubprocessParameter_whenProcessingFile_beValid(
     trufflehog_agent_file.process(scan_message_file)
     args = subprocess_check_output_mock.call_args[0][0]
 
-    assert len(args) == 5
+    assert len(args) == 4
```

Comment:
Missing unit test to diff between verified and non verified.

Reply:
[12e5213](https://github.com/Ostorlab/agent_trufflehog/pull/5/commits/12e5213d50793f97575d8917d4acd433bdb27454) Added.